src/components/shared/CustomTable.tsx

```diff
@@ -0,0 +1,46 @@
+import { Table, ScrollArea } from "@mantine/core";
+import {
+  TH_TABLE_CLIENTS,
+  TR_TABLE_CLIENTS,
+} from "../../constants/clients/clients";
+
+import { CustomTableTH } from "../../styles/CustomTable";
+
+const CustomTable = () => {
+  const headerTable = TH_TABLE_CLIENTS.map((thName: string, index: number) => {
+    return <CustomTableTH key={index}>{thName}</CustomTableTH>;
+  });
+
+  const rowsTable = TR_TABLE_CLIENTS.map((td: any, index: number) => {
+    return (
+      <Table.Tr key={index}>
+        {Object.keys(td).map((key: string, index: number) => {
+          return (
+            <Table.Td key={index} align="center">
+              {td[key]}
+            </Table.Td>
+          );
+        })}
```

Comment:
what if each cell has its own logic?
do not use index for key
```suggestion
        {Object.keys(td).map((key: string, index: number) =>
            <Table.Td key={index} align="center">
              {td[key]}
            </Table.Td>
          )}
```